.github/labeler.yml

```
@@ -20,7 +20,7 @@ pre-commit:
   - .pre-commit-config.yaml
   - .pre-commit-hooks.yaml
 python:
-  - any: ["*.py"]
+  - "spectrafit/**/*.py"
```

Not working -> Switch back to `any`
```suggestion
  - any: ["*.py"]
```